ports/libfido2/vcpkg.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "libfido2",
+  "version-string": "1.7.0",
```

```suggestion
  "version": "1.7.0",
```
It might be better to use `version `here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions